src/braket/ir/openqasm/program_v1.py

```diff
@@ -32,3 +38,9 @@ class Program(BraketSchemaBase):
     _PROGRAM_HEADER = BraketSchemaHeader(name="braket.ir.openqasm.program", version="1")
     braketSchemaHeader: BraketSchemaHeader = Field(default=_PROGRAM_HEADER, const=_PROGRAM_HEADER)
     source: str
+    inputs: Optional[
```

Comment:
Are these related to `input` and `output` keywords in OQ3? What is the expected usage for the values here?

Reply:
Yes, exactly. The demos in the simulator PR show how these are used